docs/kubernetes/ingress-usage.md

```diff
@@ -76,6 +76,52 @@ Cloud migration.
               serviceName: app-svc
               servicePort: 80
 
+### Multiple Ingresses defining the same route
+
+!!! Warning
+    
+    If multiple backends define the same `Host` header, traffic routing may become 
+    non-deterministic. 
```

Comment:
This should rather be if multiple ingresses define the same host and the same predicates.